From e0145529893b9ff57011f4cdbdc46ad029091e5b Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 14 Dec 2017 15:08:13 +0800
Subject: [PATCH] ARM: dts: rockchip: remove the psci on rk3036 kylin

As the kylin had not ready supported it during the short time.
Due to kylin enabled the psci will cause crash on bootup.

..
[    0.000000] psci: probing for conduit method from DT.
[    0.000000] Bad mode in prefetch abort handler detected
[    0.000000] Internal error: Oops - bad mode: 0 [#1] SMP ARM
[    0.000000] Modules linked in:
[    0.000000] CPU: 0 PID: 0 Comm: swapper Not tainted 4.4.103 #294
[    0.000000] Hardware name: Generic DT based system
[    0.000000] task: c130d040 task.stack: c1300000
[    0.000000] PC is at 0xa60656a8
[    0.000000] LR is at __invoke_psci_fn_smc+0x38/0x44
[    0.000000] pc : [<a60656a8>]    lr : [<c0922234>]    psr: 600001d6

Change-Id: I2edc8d1becc63e932321698b6d4a6090b527a360
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm/boot/dts/rk3036-kylin.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/rk3036-kylin.dts b/arch/arm/boot/dts/rk3036-kylin.dts
index 1d1f991538ef..ec93e8b4a524 100644
--- a/arch/arm/boot/dts/rk3036-kylin.dts
+++ b/arch/arm/boot/dts/rk3036-kylin.dts
@@ -62,6 +62,8 @@
 		};
 	};
 
+	/delete-node/ psci;
+
 	sdio_pwrseq: sdio-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		pinctrl-names = "default";
-- 
2.35.3

